evolver/app/routers/experiment.py

```diff
@@ -20,7 +20,21 @@ def get_experiment_logs(request: Request, experiment_name: str):
 
 @router.get("/{experiment_name}")
 def get_experiment_overview(request: Request, experiment_name: str):
+    evolver = request.app.state.evolver
+    experiment = evolver.experiments[experiment_name]
+
+    # Extract controller configs and current states
+    controller_data = []
+    for controller in experiment.controllers:
+        controller_info = {
+            "name": controller.name,
+            "type": controller.__class__.__name__,
+            "config": controller.config_model.model_dump(),
+        }
+        controller_data.append(controller_info)
+
     return {
-        "config": request.app.state.evolver.experiments[experiment_name],
+        "config": experiment,
         "logs": get_experiment_logs(request, experiment_name),
+        "controllers": controller_data,
```

Comment:
likewise, I think this information is available in config above, it is structured as:

```
config:
  controllers: 
    - ...
```